File: base-forge-security-authentication-spring-boot-starter-autoconfigure/src/main/java/com/gewuyou/baseforge/security/authentication/autoconfigure/config/entity/SecurityAuthenticationProperties.java
package com.gewuyou.baseforge.security.authentication.autoconfigure.config.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.boot.context.properties.ConfigurationProperties;

/**
 * 安全性身份验证属性
 *
 * @author gewuyou
 * @since 2025-01-15 22:07:46
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
@ConfigurationProperties("base-forge.security.authentication")
public class SecurityAuthenticationProperties {
    /**
     * 基础地址
     */
    private String baseUrl = "/auth/c";
    /**
     * 登录地址
     */
    private String loginUrl = "/login";
    /**
     * 登出地址
     */
    private String logoutUrl = "/logout";
}

File: base-forge-security-authentication-spring-boot-starter-autoconfigure/src/main/java/com/gewuyou/baseforge/security/authentication/autoconfigure/config/entity/SecurityAuthenticationProperties.kt
package com.gewuyou.baseforge.security.authentication.autoconfigure.config.entity

import org.springframework.boot.context.properties.ConfigurationProperties

/**
 * 安全性身份验证属性
 *
 * @author gewuyou
 * @since 2025-01-03 10:12:23
 */
@ConfigurationProperties("base-forge.security.authentication")
data class SecurityAuthenticationProperties(
    /**
     * 登录基础地址
     */
    var baseLoginUrl: String = "/auth/login",

    /**
     * 普通登录页面地址(用户唯一标识符：密码登录)
     */
    var normalLoginUrl: String = "/normal",

    /**
     * 登出页面地址
     */
    var logoutUrl: String = "/logout",
)
